### PR TITLE
Support TXT, SRV and PTR records

### PR DESCRIPTION
**Pull request by NN---**

### CODE REVIEW

File: network/dns_packet.ksy [hunk 1]
```diff
@@ -144,6 +148,29 @@ types:
...
+      - id: target
+        type: domain_name
+  txt:
+    seq:
+      - id: length
+        type: u1
+      - id: text
+        type: str
+        size: length
+        encoding: UTF-8
```
Comment (GreyCat) [inline network/dns_packet.ksy:168]: Are we sure that this comes in UTF-8? My guess would be that DNS protocol longs before predates Unicode, so likely it's treated as raw byte array?

Comment (NN---) [reply network/dns_packet.ksy:168]: Good point.
Let me double check.

Comment (NN---) [reply network/dns_packet.ksy:168]: https://tools.ietf.org/html/rfc1464

RFC says ASCII.
I’ll check whether there is a update to this RFC.

Comment (NN---) [reply network/dns_packet.ksy:168]: I split dns into dns and mdns packet.
It is better to have on dns_packet with UTF-8 if it is possible.

Comment (GreyCat) [reply network/dns_packet.ksy]: Let's stick with UTF-8 for now then, and we can always make it more strict or more relaxed later.

**Comment by GreyCat** [comment]
Merging this, thanks for your contribution!

**Pull request merged**